odoo/__init__.py

```diff
@@ -16,7 +16,8 @@
 ]
 
 import sys
-assert sys.version_info > (3, 7), "Outdated python version detected, Odoo requires Python >= 3.7 to run."
+min_py_version = (3, 7)
+assert sys.version_info > min_py_version, f"Outdated python version detected, Odoo requires Python >= {'.'.join(map(str, min_py_version))} to run."
```

Comment:
nitpicking
```suggestion
MIN_PY_VERSION = (3, 7)
assert sys.version_info > MIN_PY_VERSION, f"Outdated python version detected, Odoo requires Python >= {MIN_PY_VERSION} to run."
```
(if you are not able to start your server, how is formated the version is not that important